### PR TITLE
patch debug url redirect

### PR DESCRIPTION
**Pull request by oda-alexandre**

### CODE REVIEW

File: htdocs/core/tpl/login.tpl.php [hunk 1]
```diff
@@ -91,7 +91,7 @@
 <div class="login_center center"<?php print empty($conf->global->MAIN_LOGIN_BACKGROUND)?' style="background-size: cover; background-position: center center; background-attachment: fixed; background-repeat: no-repeat; background-image: linear-gradient(rgb('.$colorbackhmenu1.',0.3), rgb(240,240,240));"':'' ?>>
 <div class="login_vertical_align">
 
-<form id="login" name="login" method="post" action="<?php echo $php_self; ?>">
+<form id="login" name="login" method="post" action="<?php echo DOL_URL_ROOT . $php_self; ?>">
```
Comment (hregis) [inline htdocs/core/tpl/login.tpl.php:94]: @oda-alexandre 
$php_self contains DOL_URL_ROOT ?!

File: htdocs/core/tpl/passwordforgotten.tpl.php [hunk 2]
```diff
@@ -69,7 +69,7 @@
 <div class="login_center center"<?php print empty($conf->global->MAIN_LOGIN_BACKGROUND)?' style="background-size: cover; background-position: center center; background-attachment: fixed; background-repeat: no-repeat; background-image: linear-gradient(rgb('.$colorbackhmenu1.',0.3), rgb(240,240,240));"':'' ?>>
 <div class="login_vertical_align">
 
-<form id="login" name="login" method="POST" action="<?php echo $php_self; ?>">
+<form id="login" name="login" method="POST" action="<?php echo DOL_URL_ROOT . $php_self; ?>">
```
Comment (hregis) [inline htdocs/core/tpl/passwordforgotten.tpl.php:72]: @oda-alexandre 
$php_self contains DOL_URL_ROOT ?!

**Comment by eldy** [comment]
Does this fix a bug ? Can you describe how to reproduce it ?

**Comment by oda-alexandre** [comment]
if you install with docker the URL is lost to the page http://mysite/index.php?mainmenu=home&username=admin the patch forces the URL http://mysite/dolibarr/index.php?mainmenu=home&username=admin

https://github.com/oda-alexandre/aws-stack/blob/master/docker-compose.yml

**Comment by eldy** [comment]
The problem seems to be into docker installation.
Can you describe how you make your install with docker ?

**Pull request closed without merging**